client/signup/config/steps-pure.js

```diff
@@ -808,6 +808,17 @@ export function generateSteps( {
 		},
 
 		// Woocommerce Install steps
+		'store-address': {
+			stepName: 'store-address',
+			props: {
+				headerTitle: i18n.translate( 'Add an adress so you can get paid' ),
```

Comment:
```suggestion
				headerTitle: i18n.translate( 'Add an address so you can get paid' ),
```
Typo fix, in case it stays the same.